pandas/core/base.py

```diff
@@ -2,7 +2,7 @@
 Base and utility classes for pandas objects.
 """
 import builtins
-from collections import OrderedDict
+from collections import OrderedDict, abc
```

Comment:
minor nit. IMO shouldn't import abc from collections. to avoid confusion and potential future clashes with the abc module.

Reply:
I agree, though there are other files in the code which do that so I figured it's fine.
Changed.

Reply:
Apparently the guidelines in code_check.sh point to using `from collections import abc` instead of `from collections.abc import`, so changes reverted.

Reply:
fair enough. not sure of the reason for that.